test/integration/features/story_health.feature

```diff
@@ -22,7 +22,16 @@ Feature:
             | vm-driver |
             | libvirt   |
 
-    @darwin @linux 
+    @windows
+    Scenario: Start CRC on Windows
+        Given executing "crc setup" succeeds
+        When starting CRC with default bundle and nameserver "10.75.5.25" succeeds
```

Comment:
This nameserver looks like something which is going to be very specific to our CI?

Reply:
yes, you're right. We discussed it on Slack. Without it we're unable to see outside network.